examples/react-native/index.js

```diff
@@ -1,3 +1,6 @@
+import 'react-native-get-random-values';
+import 'react-native-url-polyfill/auto';
```

Comment:
Curious, are these needed for the current PR?

Reply:
They're required by the Amplify JS `Auth` category

Reply:
Ah! a comment would be helpful for future readers.